.travis.yml

```diff
@@ -104,8 +104,11 @@ script:
     fi
     ~/.venv/bin/tox
 
+before_install:
+  - pip install codecov
```

Comment:
This isn't running inside of a venv on OS X. I'd just leave this where the `pip install coveralls` was.